.gitignore

```diff
@@ -1,8 +1,8 @@
-# DotNet7
-**/bin
-**/obj
-**/.vs
-.vs/
+build/
+publish/
 
-# Rider
```

Comment:
The section names here aren't accurate.
Bin & obj are .NET specific but not .NET 7 specific.
.vs is Visual Studio specific, not .NET.
I opted to remove all section headers here because we have so few of these.